vdoctl/cmd/delete.go

```diff
@@ -58,6 +58,8 @@ Currently the command supports vanilla k8s cluster`,
 				cobra.CheckErr(fmt.Errorf("Error occurred when deleting VDO,  %v", err))
 			}
 		}
+
+		fmt.Println("Since you have invoked delete VDO command, eventually the deployments will be deleted.")
```

Comment:
lets word this as 
This command deletes the VDO operator deployment eventually the operator pods will be deleted

Reply:
is it possible to modify the delete command to vdoctl delete vdo